src/partition/FileUtils.h

```diff
@@ -46,8 +48,11 @@ class InputFile {
     /// Return the path of the input file.
     boost::filesystem::path const &path() const { return _path; }
 
+    // Needed in derived class InputFileArrow
+    virtual int getBatchNumber() const {return -1;}
```

Comment:
Problem with code formatting within curly brackets `{` ... `}`. Please, run this tool on your code before each commit:
```
qserv build -j24 --no-cmake --no-make --no-build-image --clang-format=REFORMAT
```